src/dawn/Optimizer/PassComputeStageExtents.cpp

```diff
@@ -56,6 +61,11 @@ bool PassComputeStageExtents::run(
         // check which (previous) stage computes the field (read in fromStage)
         for(int j = i - 1; j >= 0; --j) {
           Stage& toStage = *(stencil.getStage(j));
+///////////////////////////////
+/*
+          if(!readInterval.overlaps(toStage.interval()))
```

Comment:
remove?

Reply:
This is the overly greedy algorithm. Should we comment on it here?